codex-rs/tui/src/chatwidget.rs

```diff
@@ -1220,6 +1224,13 @@ impl ChatWidget {
         self.add_to_history(history_cell::new_mcp_tools_output(&self.config, ev.tools));
     }
 
+    fn on_list_custom_prompts(&mut self, ev: ListCustomPromptsResponseEvent) {
+        let len = ev.custom_prompts.len();
+        debug!("received {} custom prompts", len);
```

Comment:
I believe our stricter lint checks should reject this now? Maybe it doesn't know about `tracing::debug()`.

```suggestion
        debug!("received {len} custom prompts");
```